aws/common/file_scanning.tf

```
@@ -0,0 +1,8 @@
+module "s3_scan_objects" {
+  source = "github.com/cds-snc/terraform-modules?ref=v3.0.9//S3_scan_object"
```

You should be able to use the latest release of this module:
```suggestion
  source = "github.com/cds-snc/terraform-modules?ref=v5.0.0//S3_scan_object"
```